README.md

```diff
@@ -1,24 +1,201 @@
-# README
+<a name="readme-top"></a>
```

Comment:
_note: I couldn't find an open PR in the frontend project for me to review.  Therefore, I'll include my comments here.  I'll base my frontend review on the [main branch on that project,](https://github.com/Ezzywealth/cab_bookings_frontend/tree/main) since that's the one with more commits and includes a more complete readme file._

# Frontend review

- [ ] The frontend project requires one or more `.env` files in order to be run correctly locally.  This means there are special procedures to run your project as intended.  I couldn't find instructions or mentions about this matter (I looked in the readme file, review request, and PR description).

![image](https://github.com/Ezzywealth/cab_bookings_api/assets/57421823/9ea73130-001f-4e9b-b672-dc6af35cce22)

![image](https://github.com/Ezzywealth/cab_bookings_api/assets/57421823/b7a88df1-3e0f-47ae-b972-3a77d300a136)

![image](https://github.com/Ezzywealth/cab_bookings_api/assets/57421823/a143b565-7a98-4f8f-986a-e4f62886e4a4)

Kindly, add specific instructions on your readme file about what needs to be done to run your project 

_note:  Since I cannot access the frontend, I cannot test its functionality and styling.  This will be reviewed at the next review._

- [ ]  Please, kindly refactor the main `App component to ES6 arrow function, so your code complies with the latest standards.

![image](https://github.com/Ezzywealth/cab_bookings_api/assets/57421823/57793685-8ea6-4b4e-885d-e42a595c8fbb)


Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪 

- [ ] Kindly, remove the old commented code found on these files:

- App.js
- NewReservation.test.js
- AddCab.jsx

That way, the repo's cleanliness will be improved 👍